pom.xml

```diff
@@ -20,14 +21,20 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.7</source>
+          <target>1.7</target>
```

Comment:
Java 8 would be more comfortable, but would lock out updates to `script-security` to Jenkins versions prior to 2.60.x.